servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/FlushStrategyOnServerTest.java

```diff
@@ -96,42 +105,55 @@
         }
     }
 
-    public FlushStrategyOnServerTest(final Param param) throws Exception {
-        interceptor = new OutboundWriteEventsInterceptor();
-        channel = new EmbeddedDuplexChannel(false, interceptor);
-        useAggregatedResponse = new AtomicBoolean();
-        StreamingHttpService service = (ctx, request, responseFactory) -> {
+    public FlushStrategyOnServerTest(final Param param) {
+        this.param = param;
+        this.interceptor = new OutboundWriteEventsInterceptor();
+        this.useAggregatedResponse = new AtomicBoolean();
```

Comment:
instead of using shared state and having to worry about concurrency etc. consider just setting a header in the request and reading it on the server.